tools/upgrade_versions.py

```diff
@@ -28,6 +28,7 @@
 from lib import rpm_package
 from lib.utils import replace_str_in_file
 from lib.versions_repository import setup_versions_repository
+from tools.update_versions_readme import update_versions_in_readme
```

Comment:
I believe reusable functions should be placed in `lib`, so we wouldn't have any `import tools`